apps/mail/components/context/command-palette-context.tsx

```
@@ -1405,10 +1406,10 @@ export function CommandPalette({ children }: { children: React.ReactNode }) {
     <>
       <div className="flex items-center border-b px-3">
         <button
-          className="text-muted-foreground hover:text-foreground mr-2"
+          className="text-muted-foreground hover:text-foreground ml-2"
           onClick={() => setCurrentView('filter')}
         >
-          ←
+          <ArrowLeft className="h-4 w-4"/>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)